corehq/apps/translations/tasks.py

```diff
@@ -93,11 +103,13 @@ def pull_translation_files_from_transifex(domain, data, email=None):
             email = EmailMessage(
                 subject='[{}] - Transifex pulled translations'.format(settings.SERVER_ENVIRONMENT),
                 body="PFA Translations pulled from transifex.",
-                to=[email],
+                to=[user_email],
                 from_email=settings.DEFAULT_FROM_EMAIL
             )
             email.attach(filename=filename, content=file_obj.read())
             email.send()
+    except Exception as e:
+        notify_error(e)
```

Comment:
I'd add in a `raise` here, so we still get the error on sentry.

Reply:
thanks @esoergel , added.